convert2rhel/pkghandler.py

```diff
@@ -1144,52 +1111,147 @@ def clean_yum_metadata():
     loggerinst.info("Cached yum metadata cleaned successfully.")
 
 
-def _package_version_cmp(pkg_1, pkg_2):
-    """Compare the version key in a given package name.
+def compare_package_versions(version1, version2):
+    """Compare two EVR versions against each other, as well as
+    the name and architectures
```

Comment:
The deal of this is that we are not only comparing `EVR`, right? We are comparing more types as well, even though that seems "invisible" to the caller, correct, @pr-watson?

If so, I would change this description a bit to: 

```suggestion
    """Compare two package versions against each other, including name and arch.

    This function will receive packages in any of the following formats:
        * EVR
        * ENVR
        * ENVRA
        * NVR
        * NEVR
        * NVRA
        * NEVRA
```

Reply:
Looks better, i'll add this